source/_docs/tools/dev-tools.markdown

```diff
@@ -1,13 +1,13 @@
 ---
-title: "Developer Tools"
-description: "Description of the Developer Tools."
+title: "Developer tools"
+description: "Description of the developer tools."
 ---
 
-The frontend contains a section called **Developer Tools**.
+The frontend contains a section called **Developer tools**.
```

Comment:
```suggestion
The dashboard contains a section called **Developer tools**.
```
Suggestion for later.
We are using frontend/dashboard/UI all to describe the same thing.

I think "dashboard" is the least technical of them.